packages/tasks/src/local-apps.ts

```diff
@@ -38,20 +38,29 @@ export type LocalApp = {
 			/**
 			 * And if not (mostly llama.cpp), snippet to copy/paste in your terminal
 			 */
-			snippet: (model: ModelData) => string;
+			snippet: (model: ModelData) => string | string[];
 	  }
 );
 
 function isGgufModel(model: ModelData) {
 	return model.tags.includes("gguf");
 }
 
-const snippetLlamacpp = (model: ModelData): string => {
-	return `./main \\
+const snippetLlamacpp = (model: ModelData): string[] => {
+	return [
+		`
+## Install and build llama.cpp with curl support
+git clone https://github.com/ggerganov/llama.cpp.git 
```

Comment:
could add "&&" between each instructions? not sure it's a good idea :eyes:

Reply:
no strong opinion on my side